lib/msf/core/modules/metadata/store.rb

```diff
@@ -104,7 +104,7 @@ def configure_user_store
 
   def get_user_store
     store_dir = ::File.join(Msf::Config.config_directory, "store")
-    FileUtils.mkdir(store_dir) if !::File.exist?(store_dir)
+    FileUtils.makedirs(store_dir) if !::File.exist?(store_dir)
```

Comment:
Fix broken test logic in module cache branch that causes tests to fail when parent branches are wiped